cmd/podman/stats.go

```diff
@@ -126,6 +125,10 @@ func statsCmd(c *cli.Context) error {
 	for _, ctr := range ctrs {
 		initialStats, err := ctr.GetContainerStats(&libpod.ContainerStats{})
 		if err != nil {
+			// when doing "all", dont worry about containers that are not running
+			if c.Bool("all") && errors.Cause(err) == libpod.ErrCtrRemoved || errors.Cause(err) == libpod.ErrNoSuchCtr || errors.Cause(err) == libpod.ErrCtrStateInvalid {
```

Comment:
Not for this review, but it might be nice to have a little "containerNotRunning()" type of utility function that would return true if any of these errors were hit.